debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -428,6 +429,11 @@ private static void dumpModifiers(PrintStream out, long modifiers, int modifierS
 		}
 	}
 	
+	private static boolean isJava9() {
+		String JAVA_VERSION = System.getProperty("java.version");
+		return JAVA_VERSION.startsWith("1.9.0") || JAVA_VERSION.startsWith("9");
```

Comment:
why add "1.9.0"? this isn't a supported version string.

Reply:
I don't think this is checking the correct thing, as this test checks the version of the JVM running DDR, not the version of the JVM being inspected.